src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -79,72 +82,78 @@ const NavSearchBar = () => {
+            return (
+              <div
+                className="row"
+                key={`room-${room.title}-${index}`}
+                onClick={() => setSelectedRoom(room)}
+              >
                 <div
-                  className="row"
-                  key={`room-${index}`}
-                  onClick={() => setSelectedRoom(room)}
-                >
-                  <div
-                    className="result-avatar"
-                    style={{
-                      backgroundImage: `url(${room.image_url})`,
-                    }}
-                  ></div>
-                  <div className="result-info">
-                    <div className="result-title">{room.title}</div>
-                    <div>Room</div>
-                  </div>
+                  className="result-avatar"
+                  style={{
+                    backgroundImage: `url(${room.image_url})`,
+                  }}
+                />
+                <div className="result-info">
+                  <div className="result-title">{room.title}</div>
+                  <div>Room</div>
                 </div>
-              );
-            })}
-            {searchResult.events.map((event, index) => {
-              return (
-                <div className="row" key={`event-${index}`}>
-                  <div>
-                    <div>{event.name}</div>
-                    <div>Event</div>
-                  </div>
+              </div>
+            );
+          })}
+
+          {searchResult.events.map((event) => {
+            return (
+              <div className="row" key={`event-${event.id ?? event.name}`}>
+                <div>
+                  <div>{event.name}</div>
+                  <div>Event</div>
                 </div>
-              );
-            })}
-            {searchResult.users.map((user, index) => {
-              return (
+              </div>
+            );
+          })}
+
+          {searchResult.users.map((user) => {
+            return (
+              <div
+                className="row"
+                key={`user-${user.id}`}
+                onClick={() => setSelectedUserProfile(user)}
+              >
                 <div
-                  className="row"
-                  key={`room-${index}`}
-                  onClick={() => setSelectedUserProfile(user)}
-                >
-                  <div
-                    className="result-avatar"
-                    style={{
-                      backgroundImage: `url(${user.pictureUrl})`,
-                    }}
-                  ></div>
-                  <div className="result-info">
-                    <div key={`user-${index}`}>{user.partyName}</div>
-                  </div>
+                  className="result-avatar"
+                  style={{
+                    backgroundImage: `url(${user.pictureUrl})`,
+                  }}
+                />
+                <div className="result-info">
+                  <div>{user.partyName}</div>
```

Comment:
There is no need to set the `key` prop on an element unless it is the first child component of a `map` or similar. I assume this was leftover from older code before we wrapped it in the other `div`.